webpack.config.js

```diff
@@ -27,8 +27,9 @@ module.exports = {
     }${
       process.env.OPTIMIZE_MINIMIZE ? '.min' : ''
     }.js`,
-    library: `${!process.env.LIBRARY ? '' : process.env.LIBRARY}`,
-    libraryTarget: 'umd',
+    library: 'Keen',
```

Comment:
iiinteresting. One of the temporarily-put-on-hold changes was this https://github.com/ProdPerfect/prodperfect-keen-tracking.js/pull/13/commits/d5aee4e5a5a135e268dbcfec2a4b492f4490ae49#diff-813adaeac6dc5367589a4a6f521afef2R104, which changed the lib name so that customers who already had Keen Tracking loaded on their site could load our library without conflict. I wonder if I'll need to update this line as well in the future

Reply:
If that's something we're trying to do, there's no reason we need to call this `Keen`. It could be `ThisKeenSendsDataToWilsonBecauseWeLessThanThreeHim`

Reply:
It looks like the first few lines of the loader script does this? Am I misreading?

```
!function(name, path, ctx) {
    var latest, prev = name !== 'Keen' && window.Keen ? window.Keen : false;
    ctx[name] = ctx[name] || {
        ready: function(fn) {
            var h = document.getElementsByTagName('head')[0], s = document.createElement('script'), w = window, loaded;
            s.onload = s.onreadystatechange = function() {
                if ((s.readyState && !(/^c|loade/.test(s.readyState))) || loaded) {
                    return
                }
                s.onload = s.onreadystatechange = null;
                loaded = 1;
                latest = w.Keen;
                if (prev) {
                    w.Keen = prev
                } else {
                    try {
                        delete w.Keen
                    } catch (e) {
                        w.Keen = void 0
                    }
                }
                ctx[name] = latest;
                ctx[name].ready(fn)
            }
            ;
            s.async = 1;
            s.src = path;
            h.parentNode.insertBefore(s, h)
        }
    }
}('ProdPerfectKeen', '##############', this);

```